common/src/main/java/org/valkyrienskies/mod/mixin/feature/explosions/MixinExplosion.java

```diff
@@ -49,6 +60,50 @@ public abstract class MixinExplosion {
     @Inject(at = @At("TAIL"), method = "explode")
     private void afterExplode(final CallbackInfo ci) {
         if (isModifyingExplosion) {
+            // Custom forces
+            final Vector3d originPos = new Vector3d(this.x, this.y, this.z);
```

Comment:
Can we move this pushing code into a separate private method at least (make the private method @Unique)